administrator/components/com_content/Helper/ContentHelper.php

```diff
@@ -348,4 +348,34 @@ function ($var) use ($pk)
 			)
 		);
 	}
+
+	/**
+	 * Method to change state of multiple ids
+	 *
+	 * @param   int  $pks         Array of IDs
+	 * @param   int  $condition  Condition of the workflow state
+	 *
+	 * @return  boolean
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public static function updateContentState($pks, $condition)
+	{
+		try
+		{
+			$db    = \JFactory::getDbo();
+			$query = $db->getQuery(true);
+
+			$query->update($db->qn('#__content'))
+				->set($db->qn('state') . '=' . (int) $condition)
+				->where($db->qn('id') . ' IN (' . implode(', ', $pks) . ')');
```

Comment:
Is it for 100% sure, that we have an array here?

Reply:
yes, because this is called from the batch operation. List of selected items get set in $pks

Reply:
Perhaps we should at least check, if there is any value, otherwise the query fails:

```!empty($pks)```

Reply:
done